openmetadata-ui/src/main/resources/ui/src/styles/variables.less

```diff
@@ -34,7 +34,7 @@
 @yellow-5: #fff8e6;
 @yellow-6: #c18100;
 @yellow-7: #e7b85d;
-@error-light-color: #e5493740;
+@error-light-color: #ff9a8e40;
```

Comment:
@Ashish8689 , let's scan all the places where we are using this color variable.

Reply:
@Sachin-chaurasiya we are using this color in only searchIndexStatsTable and have verified with @Rupchavan for same!